OpenSim/Actuators/Millard2012AccelerationMuscle.h

```diff
@@ -51,7 +51,24 @@ The parent class, Muscle.h, provides
 
 namespace OpenSim {
 
+//==============================================================================
+//                   Millard2012AccelerationMuscle Exceptions
+//==============================================================================
+class Millard2012AccelerationMuscleInvalidFlag : public Exception {
```

Comment:
I think it would be incorrect to replace `[baseClassName]InvalidFlag` with `MuscleCannotEquilibrate`. The `[baseClassName]InvalidFlag` exception is thrown in a condition that can be reached only if the concrete class developer made a programming error such as

  1. Assigning a value other than 0, 1, or 2 to the first element of the SimTK::Vector that is returned by `initMuscleState()` (Millard2012AccelerationMuscle and Thelen2003Muscle) or `estimateMuscleFiberState()` (Millard2012EquilibriumMuscle); or
  2. Neglecting to handle one of the valid return values in the `switch` statement in `computeInitialFiberEquilibrium()`.

Neither of these errors relate to whether an equilibrium can be found, which I believe is what the `MuscleCannotEquilibrate` exception implies. In this situation, I believe a "fatal internal error encountered"-type of exception should be thrown (as defined on lines 57–67).

I am implementing a change that will remove the condition altogether to avoid further discussion about which exception should be thrown and what it should say.